backend/pkg/routing/routing.go

```diff
@@ -64,3 +67,37 @@ func AutogenOptionsRoutes(router *gin.Engine, gen HttpOptionsGenerator) {
 	}
 
 }
+
+func handleNoRoute(c *gin.Context) {
+	c.JSON(http.StatusNotFound, rest.Error{
+		Err:       "not found",
+		RequestID: requestid.FromContext(c.Request.Context()),
+	})
+}
+
+func handleNoMethod(c *gin.Context) {
+	c.JSON(http.StatusMethodNotAllowed, rest.Error{
+		Err:       "method not allowed",
+		RequestID: requestid.FromContext(c.Request.Context()),
+	})
+}
+
+func SwitchToReleaseMode() {
+	gin.SetMode(gin.ReleaseMode)
```

Comment:
It could still be handy for debugging locally to be able to override the gin mode using the environment.
```suggestion
	if mode := os.Getenv(gin.EnvGinMode); mode != "" {
		gin.SetMode(mode)
	} else {
		gin.SetMode(gin.ReleaseMode)
	}
	gin.DisableConsoleColor()
```